keep/api/routes/providers.py

```
@@ -597,7 +597,7 @@ async def install_provider_oauth2(
 def invoke_provider_method(
     provider_id: str,
     method: str,
-    method_params: dict,
+    body: dict = Body(...),
```

Ruff (0.8.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)